webrtc/track.go

```diff
@@ -21,13 +21,32 @@ type RemoteTrack interface {
 	ReadRTP() (*rtp.Packet, error)
 }
 
+// RTPParameters represents RTCRtpParameters which contains information about
+// how the RTC data is to be encoded/decoded.
+//
+// ref: https://developer.mozilla.org/en-US/docs/Web/API/RTCRtpSendParameters
+type RTPParameters struct {
+	SSRC        uint32
+	PayloadType uint16
```

Comment:
Not sure if this is necessary since Encodings already have the `PayloadType`. Since we're reusing the structure for setting, maybe we can have a pointer to webrtc.RTPCodec. 

Also, to be less awkward when it's being used for decoding. maybe rename "encoding" to "codec"? So, maybe something like,

```go
type RTPParameters struct {
  SelectedCodec *webrtc.RTPCodec
  Codecs   []*webrtc.RTPCodec
}
```

Reply:
Ah yes, RTPCodec already has PayloadType.
`encoding` is just from WebAPI. I feel ok with `codec`.

Reply:
some times a basic-type-copy is better? the pointer maybe scape to heap. will make a more-time-STW. If the STW > 10ms the fps maybe lost to 5~10 in high concurrency scenarios.

Do your favorite design, we can optimize it later

Reply:
This `Parameters/GetParameters` are basically called only once at initialization (after SDP exchange) in most case. Do you think it will be a performance bottleneck?